### ISSUE TITLE
The Changes from the snapshot "pending 1.18_experimental-snapshot-1" are undocumented

### ISSUE THREAD

**Issue by TheRexYo** [slicedlime/examples#9]
The Changes from the snapshot "pending 1.18_experimental-snapshot-1" are undocumented, and I was unable to find any source online to explain them. Would it be possible for you to make a repository for them as well?

**Comment by douira** [comment]
No. These new changes are not implemented through datapacks and as such you can't give an example of a datapack using them. See the news post: https://www.minecraft.net/de-de/article/new-world-generation-java-available-testing. Also see the "official" explanation video which also goes into detail about this: https://www.youtube.com/watch?v=6Ffu1gTEsVo

**Comment by TheRexYo** [comment]
> No. These new changes are not implemented through datapacks and as such you can't give an example of a datapack using them. See the news post: https://www.minecraft.net/de-de/article/new-world-generation-java-available-testing. Also see the "official" explanation video which also goes into detail about this: https://www.youtube.com/watch?v=6Ffu1gTEsVo

Yes, they are. Do you know how worldgen works? It's all part of the "vanilla" datapack. If you use the command /datapack list, you can clearly see that there's a default datapack. If you need any more proof, I have an exported copy of the default worldgen settings (courtesy of Minecraft's "export worldgen" setting). The point is, the settings aren't explained anywhere, but they do exist. You can edit them when you make Custom Worldgen datapacks. Please do your research next time, I really don't want to argue with people for no reason.

**Comment by TheRexYo** [comment]
Also, I didn't ask for how the worldgen settings work in vanilla. _They already explained that!_ I want to know how they work with custom biomes, because when you use the "export worldgen settings", you can clearly see that the changes are implemented through the "Multi-Source" biome generator type, which is what most people use for custom worldgen.

**Comment by TheRexYo** [comment]
If anyone who's actually going to take this seriously, I have some relevant information. In the exported worldgen settings, I noticed that so far the changed settings were as follows: 
- The overworld now uses the "Multinoise" biome source type.
- Biomes in the Multinoise biome source now have three new parameters; depth, continentalness (yes, you read that right), and erosion. Note that the depth parameter I'm mentioning is separate from the one used in biome jsons.

**Comment by TheRexYo** [comment]
@douira I'm not trying to be hostile, by the way, I just don't get why you're all freaking out because I told you that worldgen is still customizable. If possible, let's all try to stay civil. I didn't come here to start a fight, after all. I came here to find information, which is exactly why this is here. Also, you code JavaScript, right? Have you checked the code? If you did, perhaps you'd see what I'm referring to.

**Comment by TheRexYo** [comment]
@mainrs Is there something wrong? You seem to be peeved as well. Either that, or you're trying to get some kind of response out of me. Trolling perhaps? Either way, I really don't mind, but are you okay? Just say what's wrong, and we can talk it out like actual people. If you keep downvoting people randomly, that does little more than put a picture on someone else's screen. It means very little. If you have a problem, just speak. I'm listening.

**Comment by mainrs** [comment]
> I'm not trying to be hostile

I personally think that a lot of the sentences read differently.

> Please do your research next time, I really don't want to argue with people for no reason.

> My god this community is toxic, people get proven wrong and they spam dislikes. I know how it feels to be wrong, but you all really need to learn how to not be hostile randomly. What, you don't like it when people want to learn how things work? That's my business, not yours. Get a life.



> If possible, let's all try to stay civil.

What about "Get a life" is civil in your opinion then?

I'm not hear to argue. I just randomly read the message and reacted because I personally think that you didn't talk to people like you're on equal footing. That's all. Nothing more, nothing less. Maybe douria didn't know better and thought that it wasn't customizable. A simple "I think you're wrong here. I took a look at the datapack and saw..." would have been enough and would have been nicer as well :)

Anyway, it seems that you could answer your question, make sure to close the issue then :)

**Comment by SPGoding** [comment]
Slicedlime said something about this on his _personal, unofficial_ live stream :eyes:

https://clips.twitch.tv/SmellyJoyousSpiderOhMyDog-_6OsafRNQ7TCYbAe

**Comment by TheRexYo** [comment]
> > I'm not trying to be hostile
> 
> I personally think that a lot of the sentences read differently.
> 
> > Please do your research next time, I really don't want to argue with people for no reason.
> 
> > My god this community is toxic, people get proven wrong and they spam dislikes. I know how it feels to be wrong, but you all really need to learn how to not be hostile randomly. What, you don't like it when people want to learn how things work? That's my business, not yours. Get a life.
> 
> > If possible, let's all try to stay civil.
> 
> What about "Get a life" is civil in your opinion then?
> 
> I'm not hear to argue. I just randomly read the message and reacted because I personally think that you didn't talk to people like you're on equal footing. That's all. Nothing more, nothing less. Maybe douria didn't know better and thought that it wasn't customizable. A simple "I think you're wrong here. I took a look at the datapack and saw..." would have been enough and would have been nicer as well :)
> 
> Anyway, it seems that you could answer your question, make sure to close the issue then :)

Thanks for understanding, sorry I came across as rude. I was up late and may have been a bit low in terms of filtering. Sorry! Anyhow, the reason I haven't closed it is because other people may want to know how these work (I still don't quite get what the new parameters actually do, tbh), so it might be good to add them here in a wiki page or something.

**Comment by misode** [comment]
Now that the first real snapshots are here guess this is no longer a problem, but if you still want them, here are the changes of each experimental snapshot: <https://github.com/misode/vanilla-worldgen/commits/1.18-experimental>
